src/core/thread/mle.hpp

```diff
@@ -907,6 +907,7 @@ class Mle
     NetworkData::Leader &mNetworkData;      ///< The Network Data object.
 
     LeaderDataTlv mLeaderData;              ///< Last received Leader Data TLV.
+    bool mRFDChangingPartition;             ///< Indicating whether RFD child is changing partition with its parent
```

Comment:
This seems to only indicate that the device needs new Network Data and send Data Request messages.  Should we change the name to something like `mRetrieveNewNetworkData`?